addons/crm/models/crm_team.py

```diff
@@ -723,6 +694,17 @@ def action_opportunity_forecast(self):
         action = self.env['ir.actions.actions']._for_xml_id('crm.crm_lead_action_forecast')
         return self._action_update_to_pipeline(action)
 
+    def action_open_unassigned_leads(self):
+        action = self.action_primary_channel_button()
```

Comment:
Note that action could be False, according to base 'action_primary_channel_button' implementation. We have to be careful then as method does not necessarily return an action (well it will probably be the case in real life but hey).